CONTRIBUTING.md

```diff
@@ -77,6 +77,24 @@ Cluster API uses [GitHub milestones](https://github.com/kubernetes-sigs/cluster-
 - Dates in a release are approximations and always subject to change.
 - `Next` milestone is for work that has been triaged, but not prioritized/accepted for any release.
 
+## Documentation changes
```

Comment:
Should this go after "Contributing a Patch" (or be a sub chapter of it)

Reply:
Yeah, I actually considered this initially too but ended up putting it close to CAEP section as it's also about documentation, but really it would make sense to me if both were placed after _Contributing a Patch_ and before _Releases_.

If you don't mind a slightly bigger scope for this PR I can move both the sections, or I can just move this one for now.

Reply:
developing a CAEP in my opinion is more a process than a simple PR, while documentation change IMO is a kind of PR (so the suggestion to move this near contributing a patch).
however I don't wan't to block on this. Let's see if someone else has opinions...

Reply:
Done, moved it up. Thanks!